Sources/GRPCCore/Metadata.swift

```diff
@@ -85,6 +85,17 @@ public struct Metadata: Sendable, Hashable {
   public enum Value: Sendable, Hashable {
     case string(String)
     case binary([UInt8])
+
+    /// The value as a String. If it was originally stored as a binary, the base64-encoded String version
+    /// of the binary data will be returned instead.
+    public func encoded() -> String {
```

Comment:
To align with other existing API https://developer.apple.com/documentation/foundation/data/2142853-base64encodedstring
```suggestion
    public func base64EncodedString() -> String {
```

Reply:
It's only a base64 encoded if the underlying value is `binary` so the `base64EncodedString` isn't accurate.